src/v2/Apps/ArtAppraisals/ArtAppraisalsApp.tsx

```diff
@@ -1,16 +1,176 @@
 import { MetaTags } from "v2/Components/MetaTags"
-import { Text } from "@artsy/palette"
+import {
+  Avatar,
+  Box,
+  Button,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  Join,
+  ResponsiveBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { FullBleedHeader } from "v2/Components/FullBleedHeader"
+import { cropped } from "v2/Utils/resized"
 
 export const ArtAppraisalsApp: React.FC = () => {
   return (
     <>
-      <MetaTags title="Art Appraisals" description="todo" />
+      <MetaTags
+        title="Art Appraisals and Valuations for Your Collection — Artsy"
+        description="Artsy will connect you with the best appraiser for your collection and find the best buyers for your works through auctions, private sales, and direct listings, providing white-glove service through the entire process."
+        pathname="/art-appraisals"
+      />
 
-      <>
-        <Text variant="xl" as="h1">
-          Art Appraisals
-        </Text>
-      </>
+      <Join separator={<Spacer mt={[6, 12]} />}>
+        <Header />
+        <WhyUseArtsy />
+        <MeetOurSpecialists />
+      </Join>
     </>
   )
 }
+
+const Header: React.FC = () => {
+  return (
+    <FullBleedHeader src="https://d7hftxdivxxvm.cloudfront.net?height=3197&quality=80&resize_to=fit&src=https%3A%2F%2Ffiles.artsy.net%2Fimages%2Fappraisals-header-image.png&width=2500">
+      <Flex
+        position="absolute"
+        top={-7}
```

Comment:
@dzucconi - i could not figure out how to perfectly center this perfectly inside of the FullBleedHeader without this

Reply:
I can help you with that tomorrow. Definitely not necessary :)

Reply:
Ty 👍 Loose my mind (and so much time, all the time) over stuff like this